lib/screens/config/v2/ol_crowding.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Screens.Config.V2.OLCrowding do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          station: CurrentStopName.t()
+        }
+
+  @enforce_keys [:station]
+  defstruct station: []
+
+  use Screens.Config.Struct
```

Comment:
Because `station` is a custom type, you'll need to add it to `children` here.  

```suggestion
  use Screens.Config.Struct, children: [station: CurrentStopName]
```